sdks/java/extensions/sql/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -191,6 +191,18 @@ SqlCreate SqlCreateExternalTable() :
     }
 }
 
+SqlCreate SqlCreateTableNotSupportedMessage(Span s, boolean replace) :
+{
+
+}
+{
+  <TABLE>
+
+  {
+    return new SqlCreateTableNotSupportedMessage(s.end(this), replace);
```

Comment:
is it possible to throw from here directly without creating a new class?

Reply:
Yes it is. I changed the code to throw the exception and not create a new class.